### ISSUE TITLE
Missing Page Translations

### ISSUE THREAD

**Issue by mikojimnz** [mikojimnz/DIG4104C#14]
**Describe the bug**
Pages are missing translations.

**To Reproduce**
1. Go to 'Settings > Language Preferences'
2. Click on 'Change Language'
3. Scroll down to 'Español'
4. Go to other page
5. See error

**Expected behavior**
Each page should translate based on the settings.

**Comment by mikojimnz** [comment]
Close #14
